source/puppet/4.10/hiera_config_yaml_5.md

```diff
@@ -158,14 +158,16 @@ Key     | Data type | Expected value
 `paths` | Array     | Any number of file paths. This acts like a sub-hierarchy: if multiple files exist, Hiera searches all of them, in the order in which they're written.
 `glob`  | String    | One shell-like glob pattern, which might match any number of files. If multiple files are found, Hiera searches all of them in alphanumerical order.
 `globs` | Array     | Any number of shell-like glob patterns. If multiple files are found, Hiera searches all of them in alphanumerical order (ignoring the order of the globs).
-`mapped_paths` | Array or Hash     | A fact that is a collection (array or hash) of values. Hiera expands these values to produce an array of paths.
+`mapped_paths` | Array or Hash     | An array or hash, along with a temporary variable name to represent each element of the array or hash, and a path using the temporary variable in interpolation expressions. See the section about [mapped paths][#mapped-paths] for details.
```

Comment:
Markdown syntax: if it's a direct link instead of a "reference" style link, use parentheses in the second part instead of square brackets. `[mapped paths](#mapped-paths)`